recipe/meta.yaml

```diff
@@ -108,8 +108,8 @@ requirements:
     - pexpect       >=4.9.0,<5
     # for the symlinks
     - threejs-sage
-    - notebook      >=6.4.12,<7
```

Comment:
This pin is different than the one in sage-feedstock. I guess we don't need a pin at all here.